cmd/minikube/cmd/config/profile.go

```diff
@@ -44,6 +44,10 @@ var ProfileCmd = &cobra.Command{
 		}
 
 		profile := args[0]
+		// Check whether the profile name is container friendly
+		if !config.ProfileNameValid(profile) {
```

Comment:
thank you for fixing this,
we need to also validate in start.go where we create a profile,

do you mind also putting Before and After output in the PR description ?

I suggest putting the output of this before and after:

```
minikube start -p 2myprofile
```

Reply:
Thank you for your comments. Good ideas!

Reply:
@medyagh ready for a second revision :)